### ISSUE TITLE
LWG4019 Reversing an infinite range leads to an infinite loop

### ISSUE THREAD

**Issue by jwakely** [cplusplus/papers#1813]
Could SG9 please look at https://cplusplus.github.io/LWG/issue4019 and decide if we want a change?

If it would require a paper we can ask for one and close the issue.

**Comment by dhollman** [comment]
SG9 and LWG met jointly to discuss this issue in St. Louis. We took the following polls:

---

SG9 and LWG believe this is not a defect.

|SF|F|N|A|SA|
|-|-|-|-|-|
|3|3|0|2|0|

Weak consensus

A: It's a footgun and footguns are defects

---

SG9 and LWG agree that the standard could do something to create fewer sharp edges here, and we encourage a paper exploring options.

|SF|F|N|A|SA|
|-|-|-|-|-|
|4|2|2|0|0|

Strong consensus.

**Comment by jwakely** [comment]
The meeting minutes say that the weak consensus implies that LEWG should be asked to confirm NAD, is that right? Or should we close this?

**Comment by dhollman** [comment]
> The meeting minutes say that the weak consensus implies that LEWG should be asked to confirm NAD, is that right?

I'm generally against using extra committee time on something that's very unlikely to get consensus for change—weak consensus for NAD is still a _long_ ways away from consensus for change and "here, I'll write a paper for how this should change and it will get consensus," and even if that sentiment does exist in some subset of LEWG, closing the issue as NAD now doesn't prevent someone from writing a paper. But I think the final call on that has to be made by @inbal2l

**Comment by inbal2l** [comment]
Yes, I agree with SG9's input that this would probably be best solved by a paper. 
However, as we have a (relatively small) potential fix proposed by @jwakely , and to avoid the chance of this being shelved indefinitely, I've added the issue to our Wroclaw schedule (Friday), to be presented by: @dhollman, @foonathan (optional: @jwakely). If we won't be able to agree on the proposed resolution, we'll mark it as NAD then.
Thanks for the great input, all!
